src/edu/csus/ecs/pc2/clics/API202306/EventFeedOperation.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 1989-2019 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+/**
+ * Operations for event feed
+ *
+ * @author Douglas A. Lane, PC^2 Team, pc2@ecs.csus.edu
+ */
+public enum EventFeedOperation {
```

Comment:
It does not appear to me that this class (enum) is referenced anywhere in the PC2 project.  Perhaps it should be deleted?

Reply:
I have removed it.  It is legacy.  the "`op`" property is no longer part of the notification format, which is what this class was used for.